ui/apps/platform/src/Containers/Violations/ViolationsTablePanel.tsx

```diff
@@ -235,174 +236,182 @@ function ViolationsTablePanel({
                     </Alert>
                 ))}
             </AlertGroup>
-            <ViolationsTableSearchFilter
-                searchFilter={searchFilter}
-                onFilterChange={onFilterChange}
-                onSearch={onSearch}
-                additionalContextFilter={additionalContextFilter}
-            />
+            <PageSection hasBodyWrapper={false} className="pf-v6-u-py-0">
```

Comment:
Hopefully when we do a second pass after the initial migration to PF6, we can get rid of some of these layout utility classes in favor of matching the OCP console more closely.